pkgs/applications/misc/calibre/default.nix

```diff
@@ -197,7 +197,7 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     changelog = "https://github.com/kovidgoyal/calibre/releases/tag/v${finalAttrs.version}";
     license = if unrarSupport
-              then lib.licenses.unfreeRedistributable
+              then lib.licenses.unrar
               else lib.licenses.gpl3Plus;
```

Comment:
I know you're just changing the license to use the new `unrar` one, but shouldn't this be similar to the other cases and be a list of licenses (i.e: be _both_ GPL3+ and optionally unrar)?